src/lightcurvelynx/astro_utils/zeropoint.py

```diff
@@ -5,59 +5,34 @@
 
 from lightcurvelynx.astro_utils.mag_flux import mag2flux
 
-_lsstcam_extinction_coeff = {
-    "u": -0.458,
-    "g": -0.208,
-    "r": -0.122,
-    "i": -0.074,
-    "z": -0.057,
-    "y": -0.095,
-}
-"""The extinction coefficients for the LSST filters.
-
-Values are from
-https://community.lsst.org/t/release-of-v3-4-simulations/8548/12
-Calculated with syseng_throughputs v1.9
-"""
-
-_lsstcam_zeropoint_per_sec_zenith = {
-    "u": 26.524,
-    "g": 28.508,
-    "r": 28.361,
-    "i": 28.171,
-    "z": 27.782,
-    "y": 26.818,
-}
-"""The zeropoints for the LSST filters at zenith
-
-This is magnitude that produces 1 electron in a 1 second exposure,
-see _assign_zero_points() docs for more details.
-
-Values are from
-https://community.lsst.org/t/release-of-v3-4-simulations/8548/12
-Calculated with syseng_throughputs v1.9
-"""
-
-
-# Suppress "no docstring", because we define it via an attribute.
-def magnitude_electron_zeropoint(  # noqa: D103
+
+def sky_bg_adu_to_electrons(sky_bg_adu: npt.ArrayLike, gain: float | npt.ArrayLike) -> npt.ArrayLike:
```

Comment:
I'm not sure about the type hint here. I believe that if it is `ArrayLike`, the function is supposed to use `np.asarray` or `np.asanyarray` to convert it from inputs to something you can use the `*` operator on. Also, if we do this, the output type would be `NDArray` or something similar.

Reply:
I removed the type hints and expanded the documentation to make it clearer that this can operate on more types.